frontend/src/metabase/browse/components/RecentlyViewedModels.styled.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from "@emotion/styled";
+
+import { Grid } from "metabase/ui";
+
+export const RecentlyViewedModelsGrid = styled(Grid)`
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(13rem, 1fr));
+  gap: 0.5rem;
+  margin: 0;
+  width: 100%;
+  margin-bottom: 0.5rem;
+`;
```

Comment:
I don't think you need to add any styling to the Grid component. In fact, I think adding CSS grid properties overwrites the flexbox grid it's going for: https://v6.mantine.dev/core/grid/

Reply:
I've removed `Grid` (its default styles don't work here)